.github/workflows/ci.yml

```diff
@@ -123,10 +123,30 @@ jobs:
       - uses: actions/checkout@v2
       - name: Setup
         run: sudo apt-get install complexity
+      - name: Install Uncrustify
+        run: sudo apt-get install uncrustify
       - name: Complexity
         run: |
-          find . -iname '*.c' |\
-          xargs complexity --scores --threshold=0 --horrid-threshold=55
+          COMPLEXITY_PARAMS="--scores --threshold=10 --horrid-threshold=55"
```

Comment:
You might want to add a different horrid threshold... This threshold is not enough after this change